inngest/action.go

```diff
@@ -54,6 +54,9 @@ type ActionVersion struct {
 	// actions run.
 	Response map[string]Response `json:"response"`
 
+	// Scopes defines the permissions that this action requires to execute.
+	Scopes []string `json:"scopes"`
```

Comment:
This should mirror https://github.com/inngest/monorepo/blob/develop/pkg%2Fapplogic%2Factions%2Factions.go#L40

Reply:
It does:  `Scopes` is an alias for []string, but with some internal functionality needed for the executor only.